### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by principlebrothers**
Add Mobile-Menu, Style it according to Figma design and make it responsive using JavaScript

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,67 +1,71 @@
...
             <div class="logo">
                 <a class="a_logo" href="#">My Logo</a>
             </div>
             <nav>
-                <a class="nav-links" href="#">Portfolio</a>
-                <a class="nav-links" href="#">About</a>
-                <a class="nav-links" href="#">Contact</a>
+                <a class="nav-links" href="#portfolio">Portfolio</a>
+                <a class="nav-links" href="#about">About</a>
+                <a class="nav-links" href="#contact">Contact</a>
```
Comment (koachcode) [inline index.html:22]: - PLease i suggest you fix the hamburger menu, as the `x` is not visible, please try to make it similar to that of the figma design 

##### Your design
Screenshot from 2022-06-14 11-37-08

##### Figma design
Screenshot from 2022-06-14 11-40-33

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @principlebrothers , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- No linters error :+1: 
- Good commit message :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Coe reviewer slack:
- `@kanu mike chibundu`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (principlebrothers) [reply index.html:22]: Hi @Ginohmk, I appreciate the feedback. Then again I checked the closed button on 3 different screens and it was there. Kindly take a look at the images below.
127 0 0 1_5500_FULLTIME-MICROVERSE_MICROVERSE_Portfolio-Mobile-Version_index html(iPhone 6_7_8 Plus)
127 0 0 1_5500_FULLTIME-MICROVERSE_MICROVERSE_Portfolio-Mobile-Version_index html(Samsung Galaxy A51_71)

Then again, with regards to the colour, I check for accessibility and it was low so I needed to adjust it.

**Review by Kandy-Peter — APPROVED** [review]
Hi @principlebrothers ,

# APPROVED 🟢 


Great implementation of all the requirements in this milestone. 👏
Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

#### CONGRATULATIONS! 🎉 

##  Optional suggestion

# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**